libs/application/template-api-modules/src/lib/modules/shared/data-providers/user-profile/user-profile.module.ts

```diff
@@ -11,14 +11,24 @@ import { BaseTemplateAPIModuleConfig } from '../../../../types'
 import { UserProfileService } from './user-profile.service'
 
 import { Configuration, UserProfileApi } from '@island.is/clients/user-profile'
+import { IslykillService } from '@island.is/api/domains/user-profile'
+import { IslykillApiModule } from '@island.is/clients/islykill'
 
 export class UserProfileModule {
   static register(config: BaseTemplateAPIModuleConfig): DynamicModule {
     return {
       module: UserProfileModule,
-      imports: [SharedTemplateAPIModule.register(config)],
+      imports: [
+        SharedTemplateAPIModule.register(config),
+        IslykillApiModule.register({
+          cert: config.islykill.cert,
+          passphrase: config.islykill.passphrase,
+          basePath: config.islykill.basePath,
+        }),
+      ],
       providers: [
         UserProfileService,
+        IslykillService,
```

Comment:
Should just be able to import the Islykillmodule and inject the service. Add it to export within the module if not exported.

Reply:
Done.